packages/elements-core/src/components/TryIt/TryIt.spec.tsx

```diff
@@ -247,8 +247,8 @@ describe('TryIt', () => {
       // header param
 
       const accountIdField = screen.getByLabelText('account-id') as HTMLInputElement;
-      expect(accountIdField).toHaveValue('example id');
-      expect(accountIdField.placeholder).toBe('example: example id');
+      expect(accountIdField).toHaveValue('account-id-default');
+      expect(accountIdField.placeholder).toBe('example: account-id-default');
```

Comment:
We could optionally update this to say `default` too 🤔 :

```suggestion
      expect(accountIdField.placeholder).toBe('default: account-id-default');
```

Reply:
`defaults to: xyz`?

Reply:
![image](https://user-images.githubusercontent.com/7423098/144513142-979926c1-e673-4fdc-88bb-7dddaae26aab.png)

Reply:
Done in e99e06fc78593974eeb543b53356732873599c90